src/lib/Engineer.js

```diff
@@ -0,0 +1,33 @@
+const Employee = require("./Employee");
+
+class Engineer extends Employee {
+  constructor(name, id, email, gitHub) {
```

Comment:
If you choose to change to receive an object, this would be:

```
constructor( {name, id, email, gitHub} ) {
    super(name, id, email)
```

OR if you're feeling extra fancy, we can just the `...rest` operator, to pull out the one key we need separately, and then just spread the rest of the keys that are just being sent to `super`:

```
constructor({ gitHub, ...rest }) {
    super(rest)
    this.gitHub = gitHub
```